setup.py

```diff
@@ -19,7 +19,7 @@
     package_data={'reconcile': ['templates/*.j2']},
 
     install_requires=[
-        "sretoolbox==1.1.0",
+        "sretoolbox==1.2.2",
```

Comment:
Mmm... should we use `~=1.2` instead?

Reply:
![image](https://user-images.githubusercontent.com/28562329/144995031-2bb6ea1b-adad-425c-9322-c66082030365.png)